src/webapp/pages/bulk-apply/steps/AccessStep.tsx

```diff
@@ -59,13 +61,37 @@ export const AccessStep: React.FC<MetadataSharingWizardStepProps> = ({ builder,
                 }}
             />
 
+            <ConfirmationDialog
+                isOpen={openDialog}
+                title={i18n.t("Warning")}
+                onCancel={() => setDialogOpen(false)}
+                onSave={() => {
+                    updateBuilder(builder => ({
+                        ...builder,
+                        replaceExistingSharings: true,
+                    }));
+                    setDialogOpen(false);
+                }}
+                saveText={i18n.t("Continue")}
+                cancelText={i18n.t("Go back")}
+                maxWidth={"sm"}
+                fullWidth={true}
+            >
+                You are about to use the &quot;replace&quot; strategy. Doing so will remove pre-existing permissions and
+                is therefore a potentially dangerous operation Would you like to continue?
+            </ConfirmationDialog>
+
             <h4>Advanced options</h4>
             <FormControlLabel
                 control={
                     <Switch
                         name="replace-existing-sharings"
                         checked={builder.replaceExistingSharings}
-                        onChange={setReplaceStragegy}
+                        onChange={e =>
```

Comment:
This change works, but onChange function returns (event: React.ChangeEvent<HTMLInputElement>, isReplacingExistingSharings: boolean), so it's better to maintain this logic:

```typescript
      onChange={(event: React.ChangeEvent<HTMLInputElement>, isReplacingExistingSharings: boolean) =>
                            isReplacingExistingSharings
                                ? setDialogOpen(true)
                                : setReplaceStragegy(event, isReplacingExistingSharings)
                        }
```